theme_bookstore/views/snippets/s_three_columns.xml

```diff
@@ -5,33 +5,21 @@
         <xpath expr="//section" position="attributes">
             <attribute name="class" add="o_cc1" remove="o_cc2" separator=" "/>
         </xpath>
-        <!-- Column 1 -->
-        <xpath expr="//div[hasclass('card')]" position="attributes">
-            <attribute name="class" add="border rounded shadow" separator=" "/>
-            <attribute name="style" add="border-width: 10px !important; border-color: rgb(36, 35, 39) !important; border-radius: 0px !important; box-shadow: rgb(36, 35, 39) 0px 0px 25px 0px !important;" separator=" "/>
-        </xpath>
+        <!-- Column #1 -->
         <xpath expr="//div[hasclass('card-body')]" position="replace">
```

Comment:
I know it's not on you, but it would be better to have 2 xpaths (1 for the title and 1 for the text) to avoid recreating the card body.